databaseconnect.py

```diff
@@ -187,7 +187,7 @@ def clear_table(table_name):
 
         if input("Enter 'Y' to confirm cleaning of BOTH tables: ") in ("Y", "y"):
             for table in tables_to_be_cleaned:
-                cur.execute("DELETE FROM %s", (table))
+                cur.execute("DELETE FROM '{}'".format(table))
```

Comment:
Local testing threw errors, please share a screenshot of code testing this part and running successfully.

Reply:
This happens with table names and can be solved by [using SQL composition](https://realpython.com/prevent-python-sql-injection/#using-sql-composition). @Pihu1998